src/applications/claims-status/components/ClaimsContentionList.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import { setFocus } from '../utils/page';
+
+const MAX_CONTENTIONS = 3;
+
+function ClaimsContentionList({ contentions, contentionsHeader }) {
```

Comment:
I think we can avoid having to pass in a ref to the header by defining a click handler in the parent component (in this case `ClaimDetailLayout`) and passing that as a prop instead. We can then call the prop passed down from the parent in the click handler defined here (in this case `showAdditionalContentions`). This should help reduce the coupling between the parent and this component